object-manipulation-3/main.js

```diff
@@ -65,4 +58,34 @@ function highestScore(scores) {
 
 var highestScoreIndex = playerScores.indexOf(highestScore(playerScores));
 
-console.log(players[highestScoreIndex].name);
+function tieBreaker(scores) {
+  var tieIndices = [];
+  for (var i = 0; i < scores.length; i++) {
+    for (var j = 0; j < scores.length; j++) {
+      if (i !== j) {
+        if (scores[i] === scores[j]) {
+          tieIndices.push(i);
+        }
+      }
+    }
+  }
+  return tieIndices;
+}
+
+function logTieBreaker(arr) {
+  var names = '';
+  for (var m = 0; m < arr.length; m++) {
+    if (m === arr.length - 1) {
+      names += players[arr[m]].name;
+    } else {
+      names += players[arr[m]].name + ' and ';
+    }
+  }
+  return names + ' tied!';
+}
+
+if (tieBreaker(playerScores).length === 0) {
+  console.log(players[highestScoreIndex].name);
+} else {
+  console.log(logTieBreaker(tieBreaker(playerScores)));
+}
```

Comment:
What if there's still a tie in the replay? You should probably repeat this process until you are left with only one high score.